lib/rexml/source.rb

```diff
@@ -55,6 +55,7 @@ class Source
     attr_reader :encoding
 
     module Private
+      SCANNER_RESET_SIZE = 100000
```

Comment:
How about making this customizable only in test and using small XML in test for faster test?

Reply:
This one was improved even with fixed values.
Thanks!